<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-05-11T00:39:26 EDT" name="Default suite" finished-at="2020-05-11T00:43:04 EDT" duration-ms="218021">
    <groups>
    </groups>
    <test started-at="2020-05-11T00:39:26 EDT" name="Default test" finished-at="2020-05-11T00:43:04 EDT" duration-ms="218021">
      <class name="dataDrivenTestPackage.OrangeHRMLoginTest1">
        <test-method is-config="true" signature="setup()[pri:0, instance:dataDrivenTestPackage.OrangeHRMLoginTest1@5f16132a]" started-at="2020-05-11T00:39:27 EDT" name="setup" finished-at="2020-05-11T00:40:29 EDT" duration-ms="61895" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="orangeHRMLoginTest()[pri:0, instance:dataDrivenTestPackage.OrangeHRMLoginTest1@5f16132a]" started-at="2020-05-11T00:40:49 EDT" name="orangeHRMLoginTest" finished-at="2020-05-11T00:41:41 EDT" duration-ms="51417" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at dataDrivenUtilities.ExcelUtiliyArrayList.readExcelData(ExcelUtiliyArrayList.java:40)
at dataDrivenTestPackage.OrangeHRMLoginTest1.orangeHRMLoginTest(OrangeHRMLoginTest1.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orangeHRMLoginTest -->
        <test-method is-config="true" signature="tearDoen()[pri:0, instance:dataDrivenTestPackage.OrangeHRMLoginTest1@5f16132a]" started-at="2020-05-11T00:41:52 EDT" name="tearDoen" finished-at="2020-05-11T00:42:06 EDT" duration-ms="14049" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDoen -->
      </class> <!-- dataDrivenTestPackage.OrangeHRMLoginTest1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
